Dockerfile

```diff
@@ -1,11 +1,15 @@
 ARG SECURITY_PROFILE=generic
-
-FROM quay.io/luet/base:0.35.2 AS luet
+FROM quay.io/kairos/packages:luet-utils-0.35.2 AS luet
+FROM quay.io/kairos/packages-arm64:luet-utils-0.35.2 AS luet-arm64
 
 # Common packages for all images
 FROM alpine AS base
+ARG TARGETARCH
 ENV LUET_NOLOCK=true
-COPY --from=luet /usr/bin/luet /usr/bin/luet
+COPY --from=luet /usr/bin/luet /tmp/luet
+COPY --from=luet-arm64 /usr/bin/luet /tmp/luet-arm64
+RUN if [ "$TARGETARCH" = "arm64" ]; then mv /tmp/luet-arm64 /usr/bin/luet; else mv /tmp/luet /usr/bin/luet; fi
```

Comment:
I'd rather use different targets instead of if statements like here https://github.com/kairos-io/kairos-framework/blob/release/v2.7.41/Dockerfile#L46

Reply:
well yes, thats easy for you to say becuase you know how this docker thingies work :D

I have no idea on how to use that without breaking it :D

Reply:
ummm, want to have a look at this? I have no idea how to do it properly. Yeah I can get a container per arch but then no idea as this doesnt work:

```
ARG TARGETARCH
FROM quay.io/kairos/packages:luet-utils-0.35.2 AS luet-${TARGETARCH}

ARG TARGETARCH
FROM quay.io/kairos/packages-arm64:luet-utils-0.35.2 AS luet-${TARGETARCH}

FROM alpine as base
ARG TARGETARCH
ENV LUET_NOLOCK=true
COPY --from=luet-${TARGETARCH} /usr/bin/luet /usr/bin/luet
```

Reply:
@Itxaka the dinamic part can be used on the FROM section, the AS section needs to stay fixed, pushed a suggestion